### PR TITLE
node: Add missing links to the Network Parameters page

### PR DESCRIPTION
**Pull request by anzoman**
Adding missing links to the Network Parameters page in `docs/node`.

### CODE REVIEW

File: docs/node/genesis-doc.md [hunk 1]
```diff
@@ -30,9 +30,6 @@ Up to date information about the current genesis file and the current genesis
 document's hash can be found on the Network Parameters page ([Mainnet],
 [Testnet]).
 
-[Mainnet]: mainnet/README.md
-[Testnet]: testnet/README.md
-
 :::
```
Comment (matevz) [inline docs/node/genesis-doc.md:33]: We usually define the references at the end of the section where they occur for the first time.

```suggestion
:::

[Mainnet]: mainnet/README.md
[Testnet]: testnet/README.md
```

File: docs/node/genesis-doc.md [hunk 2]
```diff
@@ -46,6 +43,9 @@ Other Oasis networks (e.g. [Testnet]) might use different values.
 
 :::
 
+[Mainnet]: mainnet/README.md
+[Testnet]: testnet/README.md
+
```
Comment (matevz) [inline docs/node/genesis-doc.md]:
```suggestion
```

**Review by matevz — APPROVED** [review]
One nit, otherwise LGTM. Thanks!

File: docs/node/genesis-doc.md (hunk 1 above)
Comment (anzoman) [reply docs/node/genesis-doc.md:33]: Ok, updated.

File: docs/node/genesis-doc.md (hunk 2 above)
Comment (anzoman) [reply docs/node/genesis-doc.md]: Moved up.

**Pull request merged**